Library/Homebrew/cmd/update.sh

```diff
@@ -242,6 +242,15 @@ EOS
   INITIAL_BRANCH="$(git symbolic-ref --short HEAD 2>/dev/null)"
   if [[ "$INITIAL_BRANCH" != "$UPSTREAM_BRANCH" && -n "$INITIAL_BRANCH" ]]
   then
+
+    if [[ -z "$HOMEBREW_DEVELOPER" ]]
+    then
+      opoo <<EOS
```

Comment:
Let's just go for: `echo "Checking out $UPSTREAM_BRANCH in $DIR..."`

and, either here or a sensible place below, do:
```bash
  echo "To checkout $INITIAL_BRANCH in $DIR run:"
  echo "  'cd $DIR && git checkout $INITIAL_BRANCH"
```

 Want to avoid recommending setting `HOMEBREW_DEVELOPER` for now; I've got a solution incoming for this problem.